### PR TITLE
Remove Selenium NYAPPTERM

### PR DESCRIPTION
**Pull request by flooie**
PR Removes selenium from Ny App Term and subsidiaries.

### CODE REVIEW

File: juriscraper/OpinionSiteLinear.py [hunk 1]
```diff
@@ -70,3 +70,6 @@ def _get_parallel_citations(self):
 
     def _get_summaries(self):
         return self._get_optional_field_by_id("summary")
+
+    def _get_west_citations(self):
+        return self._get_optional_field_by_id("west_citation")
```
Comment (mlissner) [inline juriscraper/OpinionSiteLinear.py:75]: Isn't this no-longer a thing?

Comment (flooie) [reply juriscraper/OpinionSiteLinear.py:75]: ugh ... no - if it is my rebase was just faulty.

Comment (flooie) [reply juriscraper/OpinionSiteLinear.py:75]: ah I see this was just missed in the previous release and not a failed rebase.

Comment (flooie) [reply juriscraper/OpinionSiteLinear.py]: actually I have no idea how that got there.

Comment (flooie) [reply juriscraper/OpinionSiteLinear.py]: dropped this random addition

**Pull request merged**